### DISCUSSION TITLE
How to save orms to parquet file

### DISCUSSION THREAD

**Discussion by KnightArthurRen** [sqlalchemy/sqlalchemy#8325] [Usage Questions] [answered]
Hi there, sorry if this has been asked before but as title says is there a way to save arms to parquet files? When I google it I got [this](https://www.cdata.com/kb/tech/parquet-python-sqlalchemy.rst) but it clearly won't work as I can't find parquet as supported dialect in SQLAlchemy 1.4 doc. I know I can do it via panda df but I want to avoid this extra step, is there a way to do it directly or are we planning to add parquet dialect support in the future?

Thank you!

**Comment by CaselIT** [comment, marked as answer]
Hi,

> is there a way to do it directly or are we planning to add parquet dialect support in the future?

Short answer is not that I know and no.
Longer answer regarding if there is a way of savings to parquet is that a dialect would be needed as you pointed out. I don't know if one exists, but the sqlalchemy maintainer have no resources at the moment to support a new dialect.

**Reply by KnightArthurRen** [reply]
ok got it thank you!
